### PR TITLE
Break Non-Damageable Items

### PR DESCRIPTION
**Pull request by Charles445**
#59 
Config and implementation to allow the breaking of items like books when disenchanted. Disabled by default.

Note that a bulk disenchanter will only remove one enchantment before breaking the item.

### CODE REVIEW

File: src/main/java/de/impelon/disenchanter/DisenchantingUtils.java [hunk 1]
```diff
@@ -558,6 +558,18 @@ public static boolean isItemStackBroken(ItemStack itemstack) {
 		return itemstack.isItemStackDamageable() && itemstack.getItemDamage() > itemstack.getMaxDamage();
 	}
 
+	/**
+	 * <p>
+	 * Returns whether the given itemstack should break after one use (for non-damageable items)
```
Comment (Impelon) [inline src/main/java/de/impelon/disenchanter/DisenchantingUtils.java]:
```suggestion
	 * Returns whether the given itemstack should break (for non-damageable items).
```

**Review by Impelon — CHANGES_REQUESTED** [review]
Look good!

Please add yourself to CONTRIBUTORS.md, if you accept the terms lied out there.

**Comment by Impelon** [comment]
I changed the branch to 1.12-update, since that is the branch for changes regarding the next update.

**Comment by Charles445** [comment]
Okay, I think that sorted it all
Little contributor conflict had me confused for a moment there

**Review by Impelon — APPROVED** [review]

**Comment by Impelon** [comment]
> Little contributor conflict had me confused for a moment there

Yes, sorry about that! 
I did not think it would cause a conflict, but thinking about it again, it makes sense now. Should have made that change earlier.

**Pull request merged**